gnomad/utils/annotations.py

```diff
@@ -90,6 +90,43 @@
 }
 
 
+def annotate_with_ht(
+    t: Union[hl.MatrixTable, hl.Table],
+    annotation_ht: hl.Table,
+    fields: Optional[List] = None,
+    annotate_cols: bool = False,
+    filter_missing: bool = False,
+) -> hl.MatrixTable:
+    """
+    Annotate a MatrixTable/Table with additional annotations from another Table.
+
+    :param t: MatrixTable/Table to be annotated.
+    :param annotation_ht: Table containing additional annotations to be joined on `t`.
+    :param fields: Optional list of fields to select from `annotation_ht` and add to `t`
+    :param annotate_cols: If True, annotate columns instead of rows. Default is False.
+    :param filter_missing: If True, filter out missing rows/cols in `t` that are not
+        present in `annotation_ht`. Default is False.
+    :return: Annotated MatrixTable.
```

Comment:
```suggestion
    :return: Annotated MatrixTable/Table.
```

if a table is the input, then the code should just return a table as well, yes?